src/extensionsConfig.js

```diff
@@ -0,0 +1,71 @@
+import _keys from 'lodash/keys';
+
+const fields = {
```

Comment:
is this an example? If yes, I would keep only something random, e.g. `example_extra_field` -> string, and keep all this for CDS.
I would not put CERN/CDS stuff in InvenioILS.

WDYT?

Reply:
This is the actual CDS config, will be moved in CDS